packages/api/src/promise/index.ts

```diff
@@ -196,8 +193,8 @@ export default class ApiPromise extends ApiBase<OnCall> implements ApiPromiseInt
     };
   }
 
-  protected onCall (method: RxFn, params: Array<any>, isSubscription: boolean = false): OnCall {
-    if (!params || params.length === 0 || !isSubscription) {
+  protected onCall (method: RxFn, params: Array<any>, isSubscription?: boolean): OnCall {
+    if (!params || params.length === 0 || isSubscription === false) {
```

Comment:
Ok, I believe it is here - since we don't actually pass `isSubscription` through everywhere, it defaults to `true` - basically what I'm seeing is that stuff doesn't actually create subscriptions. (And yes, this was an after-the-tests "o, that looks good", erm, _improvement_)